src/components/webview/sourceControl.ts

```diff
@@ -16,6 +16,13 @@ type CreatePRResponse = {
 	html_url: string;
 };
 
+type ListPRResponse = {
+	html_url: string;
+	head: {
```

Comment:
I think we don't need `head`?

Reply:
we need it to find a PR for specific branch. `head.ref` is the branch name